code/game/objects/items/weaponry.dm

```diff
@@ -340,11 +340,11 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	embedding = list("pain_mult" = 0, "jostle_pain_mult" = 0, "embed_chance" = 100, "fall_chance" = 0)
 
 /obj/item/switchblade
```

Comment:
probably out of scope for this pr, but this should be a subtype of /obj/item/melee/transforming i think